components/Map.js

```diff
@@ -0,0 +1,304 @@
+    location: "Stuttgart",
+    status: "lost",
+  },
+  {
+    id: "6",
+    name: "Compass",
+    lat: 48.516252,
+    long: 9.149923,
+    location: "Stuttgart",
+    status: "lost",
+  },
+  {
+    id: "7",
+    name: "Bottle",
+    lat: 48.474338,
+    long: 8.936196,
+    location: "Rottenburg am Neckar",
+    status: "lost",
+  },
+  {
+    id: "8",
+    name: "Camera",
+    lat: 51.31271,
+    long: 9.479746,
+    location: "Kassel",
+    status: "lost",
+  },
+  {
+    id: "9",
+    name: "Backpack",
+    lat: 48.135124,
+    long: 11.581981,
+    location: "München",
+    status: "lost",
+  },
+  {
+    id: "10",
+    name: "Camera Lens",
+    lat: 51.339695,
+    long: 12.373075,
+    location: "Leipzig",
+    status: "lost",
+  },
+  {
+    id: "11",
+    name: "Lollipop",
+    lat: 51.481468,
+    long: 51.481468,
+    location: "Halle",
+    status: "lost",
+  },
+  {
+    id: "12",
+    name: "Band Aid",
+    lat: 50.356548,
+    long: 7.59942,
+    location: "Koblenz",
+    status: "lost",
+  },
+  {
+    id: "13",
+    name: "Kids Bike",
+    lat: 52.357235,
+    long: 9.751259,
+    location: "Hannover",
+    status: "lost",
+  },
+  {
+    id: "14",
+    name: "Camera",
+    lat: 50.827847,
+    long: 50.827847,
+    location: "Chemnitz",
+    status: "found",
+  },
+  {
+    id: "15",
+    name: "Backpack",
+    lat: 51.058833,
+    long: 13.728651,
+    location: "Dresden",
+    status: "found",
+  },
+  {
+    id: "16",
+    name: "Diamonds",
+    lat: 50.122427,
+    long: 8.685281,
+    location: "Frankfurt am Main",
+    status: "found",
+  },
+  {
+    id: "17",
+    name: "One Shoe",
+    lat: 50.060754,
+    long: 8.237295,
+    location: "Wiesbaden",
+    status: "found",
+  },
+  {
+    id: "18",
+    name: "Ring",
+    lat: 52.02379,
+    long: 8.550192,
+    location: "Bielefeld",
+    status: "found",
+  },
+  {
+    id: "19",
+    name: "Key",
+    lat: 53.061355,
+    long: 8.791523,
+    location: "Bremen",
+    status: "found",
+  },
+  {
+    id: "20",
+    name: "Keyboard",
+    lat: 52.268493,
+    long: 8.076874,
+    location: "Osnabrück",
+    status: "found",
+  },
+  {
+    id: "21",
+    name: "Shoe",
+    lat: 52.379393,
+    long: 13.09791,
+    location: "Potsdam",
+    status: "found",
+  },
+  {
+    id: "22",
+    name: "Handshoe",
+    lat: 53.626806,
+    long: 11.402587,
+    location: "Schwerin",
+    status: "found",
+  },
+  {
+    id: "23",
+    name: "Camera",
+    lat: 53.850987,
+    long: 10.683004,
+    location: "Lübeck",
+    status: "found",
+  },
+  {
+    id: "24",
+    name: "Football",
+    lat: 54.308592,
+    long: 10.133887,
+    location: "Kiel",
+    status: "found",
+  },
+  {
+    id: "25",
+    name: "Golden Coins",
+    lat: 53.537206,
+    long: 8.621178,
+    location: "Bremerhaven",
+    status: "found",
+  },
+  {
+    id: "26",
+    name: "Golden Jewelry",
+    lat: 48.720324,
+    long: 11.466339,
+    location: "Ingolstadt",
+    status: "found",
+  },
+];
```

Comment:
Hi Jannik,
this is a very helpful feature for your app!
I guess it would be better to have only one source of truth for your DB. I would recommend to import it the way you did in `SearchFound.js` and work with the props. 
`import { foundItems } from "../../services/Db";`